descartes_planner/include/descartes_planner/sparse_planner.h

```diff
@@ -88,7 +88,7 @@ class SparsePlanner: public descartes_core::PathPlannerBase
   };
 
   double sampling_;
-  int error_code_;
+  mutable int error_code_;
```

Comment:
I threw mutable in here because the getPath function of this class checks some conditions and maybe writes to error_code_. I thought it wasn't too much of a violation of the semantics of const-correctness to allow the class to mutate the error-code_ variable.